lib/config/rules/graphql.js

```diff
@@ -1,8 +1,8 @@
 // uses graphql-config for schema mapping
 module.exports = {
   'graphql/capitalized-type-name': 'off',
-  'graphql/named-operations': 'error',
-  'graphql/no-deprecated-fields': 'error',
+  'graphql/named-operations': ['error', {env: 'literal'}],
```

Comment:
Curious why we didn't use [env: 'apollo'](https://github.com/apollographql/eslint-plugin-graphql#common-options) here? Is that for when Apollo Engine is used or something?

Reply:
Docs aren't super clear. The difference is that we use `.graphql` files (literal) rather than some flavour of template string queries in JS (what the other options are referring to). I had originally expected the rules to work out of the box since 'apollo' is the default and we use Apollo, but 'literal' is what actually works.

Reply:
Right, that makes a lot of sense. I suppose `apollo` in JS uses the [graphql-tag](https://github.com/apollographql/graphql-tag) library, which is why they have a separate option for it. :neat: